components/back-button/back-button.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { BackButton } from "./back-button";
+
+export default {
+  title: "Back Button",
+  component: BackButton,
+  parameters: {
+    componentSubtitle: "This button returns the user to the root page.",
+  },
+};
+
+export const backButton = () => <BackButton text={"All News"} />;
```

Comment:
Remove the curly bracers in the text prop just leave it like this text="All News"